doc/ice_in

```diff
@@ -65,18 +68,18 @@
   ustar_min = 0.0005
 /
 &domain_nml
-  block_size_x = 16
-  block_size_y = 15
+  block_size_x = 15
+  block_size_y = 20
   distribution_type = "cartesian"
   distribution_wght = "latitude"
   maskhalo_bound = .true.
   maskhalo_dyn = .true.
   maskhalo_remap = .true.
-  max_blocks = 10
+  max_blocks = 15
```

Comment:
```suggestion
  max_blocks = -1
```

In the latest [025deg_jra55do_ryf](https://github.com/ACCESS-NRI/access-om3-configs/tree/025deg_jra55do_ryf), `max_blocks` has been set to -1 to allow the code to automatically determine the optimal value.  
https://github.com/ACCESS-NRI/access-om3-configs/blob/f4b2d1e916c0444f4c4620382800306f4b797fe4/ice_in#L78

I can quickly update the 1deg config.